tests/mocks/mock_tokenizer.py

```diff
@@ -1,11 +1,13 @@
+from __future__ import annotations
 from attr import define, field
 from griptape.tokenizers import BaseTokenizer
 
 
-@define(frozen=True)
+@define()
 class MockTokenizer(BaseTokenizer):
     model: str = field(kw_only=True)
-    max_tokens: int = field(default=1000, kw_only=True)
+    max_input_tokens: int = field(default=1000, kw_only=True)
+    max_output_tokens: int = field(default=1000, kw_only=True)
 
-    def count_tokens(self, text: str) -> int:
+    def count_tokens(self, text: str | list[dict]) -> int:
```

Comment:
In the `list[dict]` case do we need to loop through the elements of the dict to determine text length?

Reply:
This was just to fix a pyright error. I think all instances of `MockTokenizer` pass a `str`.